checker/src/org/checkerframework/checker/index/IndexUtils.java

```diff
@@ -0,0 +1,106 @@
+package org.checkerframework.checker.index;
+
+import com.sun.source.tree.Tree;
+import java.util.Collections;
+import java.util.List;
+import javax.lang.model.element.AnnotationMirror;
+import org.checkerframework.checker.index.minlen.MinLenAnnotatedTypeFactory;
+import org.checkerframework.checker.index.qual.MinLen;
+import org.checkerframework.common.value.ValueAnnotatedTypeFactory;
+import org.checkerframework.common.value.qual.IntVal;
+import org.checkerframework.framework.type.AnnotatedTypeMirror;
+import org.checkerframework.javacutil.AnnotationUtils;
+
+/** A collection of utility functions used by several index checker subcheckers. */
+public class IndexUtils {
```

Comment:
By convention, we name classes "...Util" rather than "...Util**s**".  Could you undo the pluralization?